init.js

```diff
@@ -260,6 +260,9 @@ function addAdminContextMenu(target, player, uuid) {
   const playerName = getPlayerName(player, true);
   
   let tooltipHtml = `<a href="javascript:void(0);" class="banPlayerAction">${getMassagedLabel(localizedMessages.context.admin.ban.label, true).replace('{PLAYER}', playerName)}</a>`;
+  tooltipHtml += player.muted ? `<br>
```

Comment:
This isn't going to work as-is because tooltips are generated once for chat messages and not updated after that, so it won't just switch which one shows. We'd have to update it to be dynamic, though maybe I can do that.